Providers/Profile.swift

```diff
@@ -296,7 +296,7 @@ public class BrowserProfile: Profile {
         let commands = items.map { item in
             SyncCommand.fromShareItem(item, withAction: "displayURI")
         }
-        self.remoteClientsAndTabs.insertCommands(commands, forClients: clients)
+        self.remoteClientsAndTabs.insertCommands(commands, forClients: clients) >>> { self.syncManager.syncClients() }
```

Comment:
Interestingly, this is already happening.

When we invoke the extension, the first thing it does is perform a client sync to get the latest clients to choose from. We lose any sent commands at this point. Even if we weren't syncing immediately on send tab we'd _still_ be having this problem. Good catch :smile_cat: